### PR TITLE
refactor: cqrs/memory management

### PR DESCRIPTION
**Pull request by gfyrag**

### CODE REVIEW

File: components/ledger/.gitignore [hunk 1]
```diff
@@ -15,5 +15,4 @@ ledger.test
 antlr-*-complete.jar
 go.work
 go.work.sum
-ledger
```
Comment (paul-nicolas) [inline components/ledger/.gitignore:18]: Why did you remove that ? it should prevent us to push the binary, am I right ?

Comment (gfyrag) [reply components/ledger/.gitignore:18]: yes, but it was hide the "ledger" package ^^
This has probably been added by error.

**Review by paul-nicolas — APPROVED** [review]
